CHANGELOG.md

```diff
@@ -6,6 +6,9 @@ This gem is moving onto its own [Semantic Versioning](https://semver.org/) schem
 
 Prior to v1.0.0 this gem was versioned based on the `MAJOR`.`MINOR` version of RuboCop. The first release of the ezcater_rubocop gem was `v0.49.0`.
 
+## v2.1.0
```

Comment:
Is this the correct version?

Reply:
if it requires manual changes to existing migration files, I would count that as not backwards compatible, and bump major version. But, if it leaves existing migrations as is, this looks correct.